File: apzkr-pzpi-21-2-butsulin-ihor/Task1-Server/apz-backend/types/models/Transfer.go
package models

import (
	"gorm.io/gorm"
	"time"
)

type Transfer struct {
	gorm.Model
	CarID       uint      `json:"carID"`
	Car         Car       `json:"car"`
	WarehouseID uint      `json:"warehouseID"`
	Warehouse   Warehouse `json:"warehouse"`
	InDate      time.Time `json:"inDate"`
	OutDate     time.Time `json:"outDate"`
}

File: apzkr-pzpi-21-2-butsulin-ihor/Task4-MobileClient/APZMobile/app/src/main/java/com/osigram/apz2024/mobile/models/Transfer.kt
package com.osigram.apz2024.mobile.models

import kotlinx.serialization.Serializable

@Serializable
data class Transfer(val ID: ULong, val carID: ULong, val car: Car, val warehouseID: ULong, val warehouse: Warehouse, val inDate: String, val outDate: String)